app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -93,13 +96,8 @@ class MEJSPlayer {
    * @return {void}
    */
   emitSuccessEvent() {
```

Comment:
Is this function being used?

Reply:
Not yet, but I put it in a while ago in case we need to hook into it.  I tried hooking into it for this scenario, but it didn't do the trick.  I can remove it if you prefer.

Reply:
No, leave it. I like having it.